app/models/user.rb

```diff
@@ -315,17 +315,19 @@ def send_get_swapping_reminder_email
     sent_emails.create!(template: SentEmail::REMINDER_GET_SWAPPING)
   end
 
-  def send_pending_swap_reminder_email
-    # FIXME: check SentEmail to avoid duplicates
-    # return if ...
-    self.pending_swap_reminder_email = true
+  def send_pending_swap_reminder_email(other_user)
+    template = SentEmail::REMINDER_PENDING_OFFER
+    already_sent = sent_emails.where(template: template).first
```

Comment:
Completely unimportant, just for my curiosity wouldn't `sent_emails.find_by_template(template)` do the trick here?

Reply:
yes, but it doesn't read as nice :)

Reply:
I like the temporary variable, main point was about `find_by_template` instead of `.where(...).first`

Reply:
I guess I was influenced by seeing the other `SentMail.where` . Rubocop has a preference between `.find_by_X("Y")`  and `find_by(X: "Y")` and and I can't remember which way round it is.

Reply:
Both work!

Reply:
Oh sorry, you meant Rubocop.